src/main/java/eu/dissco/backend/controller/UserController.java

```diff
@@ -97,21 +97,6 @@ public ResponseEntity<Void> deleteUser(Authentication authentication,
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
 
-  @ExceptionHandler(ConflictException.class)
```

Comment:
Why get rid of these?

Reply:
I didn't get rid of these but I made them generic for all classes by putting them in the `RestResponseEntityExceptionHandler.class`